packages/components/form/__tests__/form.test.tsx

```diff
@@ -22,7 +22,7 @@ import FormItem from '../src/form-item.vue'
 import DynamicDomainForm, { formatDomainError } from '../mocks/mock-data'
 
 import type { VueWrapper } from '@vue/test-utils'
-import type { FormRules } from '@element-plus/tokens'
+import type { FormRules } from '@element-plus/components/form'
```

Comment:
```
import type { FormRules } from '../src/constants'
```

Reply:
I used absolute path in tests for all components,I think this is just fine.